test/specs/dereferenceCircular.js

```diff
@@ -0,0 +1,56 @@
+/* global RAML, describe, it */
+
+'use strict';
+
+if (typeof window === 'undefined') {
+  var raml           = require('../../lib/raml.js');
+  var chai           = require('chai');
+  var chaiAsPromised = require('chai-as-promised');
+
+  chai.should();
+  chai.use(chaiAsPromised);
+} else {
+  var raml           = RAML.Parser;
+
+  chai.should();
+}
+var schema = "{\"$schema\":\"http://json-schema.org/draft-04/schema#\",\"definitions\":{\"node\":{\"type\":\"object\",\"properties\":{\"id\":{\"type\":\"string\",\"format\":\"id\"},\"label\":{\"type\":\"string\"},\"children\":{\"type\":\"array\",\"items\":{\"type\":\"object\",\"required\":[\"id\",\"label\",\"children\"]}}}}},\"type\":\"array\"}";
```

Comment:
Perfect, I prefer single quotes. I was following the same as the dereferencing tests.